### PR TITLE
[WNMGDS-200] Add the “official government website” banner to docs site

### LINKED ISSUE #414: Consider adding the federal gov banner to design.cms.gov
Consider adding the `An official website of the United States government` banner to https://design.cms.gov/ 

From US Web Design System
https://v2.designsystem.digital.gov/components/header/
>The “official government website” banner
We recommend that all federal government sites include the “official government website” banner and a logo or site name. The banner clearly identifies your site as part of the United States government, while a logo or site name helps visitors understand who you are and what you do.

This would help with building trust as well as making sure folks know that this is a .gov site

### PR DESCRIPTION
**Pull request by line47**
### Added
Added Official gov banner to the header file 

### Screenshots
Screen Shot 2019-06-24 at 12 47 40 PM

Screen Shot 2019-06-24 at 12 47 47 PM

This closes #414

### CODE REVIEW

File: packages/docs/src/scripts/components/Header.jsx [hunk 1]
```diff
@@ -1,22 +1,110 @@
...
+                </div>
+                <div className="grid-col-fill tablet:grid-col-auto">
+                  <p className="usa-banner__header-text">
+                    An official website of the United States government
+                  </p>
+                </div>
+                <button
+                  onClick={this.handleToggleBannerButton}
+                  className="usa-accordion__button usa-banner__button"
+                  aria-expanded="false"
```
Comment (bernardwang) [inline packages/docs/src/scripts/components/Header.jsx]: Shouldn't this be dynamic too like this? `aria-expanded={this.state.isBannerOpen}`

File: packages/docs/src/styles/components/_UsaBanner.scss [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    max-width: 100%;
+}
+.grid-col-auto {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 0 auto;
+    flex: 0 0 auto;
+    width: auto;
+max-width: 100%;
+}
+.tablet\:grid-col-1, .tablet\:grid-col-auto {
```
Comment (bernardwang) [inline packages/docs/src/styles/components/_UsaBanner.scss]: I've never seen this syntax before with the backslash \, do you know what this means?

File: packages/docs/src/styles/docs.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 // Libraries
+// @import '../../../../packages/support/node_modules/uswds/src/stylesheets/uswds.scss';
```
Comment (bernardwang) [inline packages/docs/src/styles/docs.scss]: can this be removed?

**Comment by bernardwang** [comment]
I haven't looked closely into the css for the solution to this, but it looks like the banner on USWDS's site collapses to a single column at a certain point. I think it makes sense to have that responsive behavior on ours as well, heres some pictures for comparison.

CMS
Screen Shot 2019-06-24 at 5 25 46 PM
 USWDS
Screen Shot 2019-06-24 at 5 25 50 PM

File: packages/docs/src/styles/components/_UsaBanner.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    -webkit-box-flex: 0;
+    -ms-flex: 0 0 auto;
+    flex: 0 0 auto;
+}
+.tablet\:grid-col-6 {
+    width: 50%;
+}
+.usa-banner__header--expanded {
```
Comment (bernardwang) [inline packages/docs/src/styles/components/_UsaBanner.scss]: I think there should be some css here as well to make the arrow flip to point up.

File: packages/docs/src/styles/components/_UsaBanner.scss (hunk 2 above)
Comment (line47) [reply packages/docs/src/styles/components/_UsaBanner.scss]: i's a way of namespacing styles for example `tablet\:grid-col-6` renders as  `tablet:grid-col-6` which is a styles applied at tablet size which is defined as `@media (min-width: 40em)`

File: packages/docs/src/styles/docs.scss (hunk 3 above)
Comment (line47) [reply packages/docs/src/styles/docs.scss]: yep!

**Comment by line47** [comment]
Ready for a final pass @bernardwang

**Pull request merged**